src/components/Tool/FixCommitmentInfo/FixCommitmentInfo.test.tsx

```diff
@@ -84,6 +86,9 @@ const Components = ({
 describe('FixCommitmentInfo', () => {
   beforeEach(() => {
     setContactFocus.mockClear();
+    (useGetAppSettings as jest.Mock).mockReturnValue({
```

Comment:
Can we get this working by adding `<AppSettingsProvider>` to the `TestComponent` instead of mocking `useGetAppSettings`?

Reply:
Can you send me your test file that isn't working? When I remove the `useGetAppSettings` mock and add `<AppSettingsProvider>` to `Components`, the tests still pass.